src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -375,7 +375,6 @@ NuGet.Commands.LocalsCommandRunner
 NuGet.Commands.LocalsCommandRunner.ExecuteCommand(NuGet.Commands.LocalsArgs localsArgs) -> void
 NuGet.Commands.LocalsCommandRunner.LocalsCommandRunner() -> void
 NuGet.Commands.LockFileBuilder
-NuGet.Commands.LockFileBuilder.CreateLockFile(NuGet.ProjectModel.LockFile previousLockFile, NuGet.ProjectModel.PackageSpec project, System.Collections.Generic.IEnumerable<NuGet.Commands.RestoreTargetGraph> targetGraphs, System.Collections.Generic.IReadOnlyList<NuGet.Repositories.NuGetv3LocalRepository> localRepositories, NuGet.DependencyResolver.RemoteWalkContext context) -> NuGet.ProjectModel.LockFile
```

Comment:
For better or worse, this is a breaking change that we shouldn't introduce ;) 
Some of our assemblies are really tooling specific, and it would've great if the public apis were carefully crafted, but that's not the case, so we're mkaing it work with what we have right now. 

A long way of saying, please don't introduce a  breaking change.

Reply:
Done